### DISCUSSION TITLE
求助关于aria2设置

### DISCUSSION THREAD

**Discussion by tifa2007** [AlistGo/alist#2203] [Q&A]
我的aria2在AriaNg设置了启用rpc scure就是使用 https://192.168.1.2:6800/jsonrpc进行访问
但是alist这样输入会报错cannot validate certificate for *********** because it doesn't contain any IP SANs

若关掉使用 http://192.168.1.2:6800/jsonrpc可以进行访问 但是AriaNg我又连不上了

现在问下如何实现开启rpc scure 能用https 连接aria2啊

**Comment by rexhang** [comment]
不知道
